### PR TITLE
rgw_frontend_ssl_key is removed after upgrade to Pacific (bsc#1200319)

### PR DESCRIPTION
**Pull request by giubacc**
- Changed format from Note to Important
- Information added also in upgrade-to-pacific.xml

Fixes: bsc#1200319
Signed-off-by: Giuseppe Baccini <giuseppe.baccini@suse.com>

### CODE REVIEW

File: xml/upgrade-to-pacific.xml [hunk 1]
```diff
@@ -252,6 +252,24 @@
  <sect1 xml:id="upgrade-gateways-7p">
   <title>Gateway service upgrade</title>
 
+ <sect2 xml:id="upgrade-object-gateway-7p">
+ <title>Upgrading Object Gateway</title>
```
Comment (asettle) [inline xml/upgrade-to-pacific.xml:256]:
```suggestion
 <title>Upgrading the &ogw;</title>

File: xml/upgrade-to-pacific.xml [hunk 2]
```diff
@@ -252,6 +252,24 @@
...
+   <important xmlns="http://docbook.org/ns/docbook">
```
Comment (asettle) [inline xml/upgrade-to-pacific.xml:257]:
```suggestion
   <important>

**Comment by giubacc** [comment]
> Just little things

done!

**Review by asettle — APPROVED** [review]

**Review by dariavladykina — APPROVED** [review]
LVGTM

**Pull request merged**